lib/versioner/util.go

```diff
@@ -293,3 +294,51 @@ func findAllVersions(fs fs.Filesystem, filePath string) []string {
 
 	return versions
 }
+
+func CleanByDay(cleanoutDays int, folderFs fs.Filesystem, versionsFs fs.Filesystem, ctx context.Context) error {
```

Comment:
This can be package private by being lowercase.

Reply:
Also, go always has context as the first argument. I think in this case I'd have `(ctx, folderFs, versionerFs, days)` which somewhat gives them hierarcy of importance.

Reply:
Actually, folderFs is not even used.

Reply:
I want to read code as well as you. Thank you for teaching me so much. I promise to do better next time.